src/views/admin/KlasseDetailsView.vue

```diff
@@ -72,8 +72,10 @@
 
   function isFormDirty(): boolean {
     // Only check for dirtiness if the form is in edit mode
+    // Only check if klassenname is dirty, because selectedSchule is readonly
     if (!isEditActive.value) return false;
-    else return isFieldDirty('selectedSchule') || isFieldDirty('selectedKlassenname');
+    else if (selectedKlassenname.value === organisationStore.currentKlasse?.name) return false;
+    else return isFieldDirty('selectedKlassenname');
```

Comment:
Wollen wir dann auch für RolleDetailsView das gleiche machen? Da werden auch die ReadOnly Inputs berücksichtigt bei Dirtiness.

Reply:
Bei RolleDetails wird isFormDirty über die validation utils geprüft. Das benutzen wir auch für die RolleCreation, deshalb geht das nicht. Wir könnten das bei den Klassen refactorn, aber ungerne in diesem Scope